src/cc_toolchain/BUILD

```diff
@@ -69,56 +69,57 @@ filegroup(
 )
 
 filegroup(
-    name = "clang_wrappers",
-    srcs = glob(["wrapper/clang-*"]),
+    name = "k8_gcc_wrappers",
+    srcs = glob(["wrapper/k8_gcc-*"]),
```

Comment:
Are these wrappers specific to `linux_gcc`?

Reply:
yea, fixed